docs/dashboards/chart-builder.md

```diff
@@ -0,0 +1,176 @@
+
+  __Example__:
+  
+  Consider you collect 2 different metrics which have similar meaning. On one host you collect a
+  metric which is called `os.disk.usage` and on the second host you collect a metric called
+  `os.disk.used`. You'd like to show a total disk usage from 2 hosts. Unfortunately metrics
+  are called differently. Here's where `coalesce` is useful. You simply call the function 
+  ```coalesce(`os.disk.usage`, `os.disk.used`, 0)```. Depending on the metrics' name for a given
+  datapoint one of `os.disk.usage`, `os.disk.used` and `0` will be picked.
+
+
+## Filter by
+
+Filter section is used to specify a criteria that is used to find only matching datapoints.
+Suppose you collect data from multiple hosts but you'd like to see a chart containing only
+2 of them (`host1` and `host2`). You have to pick the right filter
+in [Filter by](#filterby) section. You chose tag name `os.host` and 2 tag values `host1` and `host2`.
+
+Not all tags are physically stored with the datapoints but there can be a
+logical connection between a tag (for example `os.host`) and so called tag alias. This tag
+alias can describe for example availability zone, cpu type, etc. Read more
+about [tag aliases](/monitoring/tag-aliases-support).
+
+You can filter by tag aliases the same way as they are regular tags so this is completely
+transparent to you.
+
+You can use [variables](#variables) as tag values so they can be configured dynamically
+from the report level.
+
+## Group by
+
+In Group by section you choose what grouping data series on a graph. For example, if you
+choose `os.host` tag and aggregation in `Aggregation` field you choose `all separately`
+the chart will show data series for each host separately. Each data series is made up of
+the selected metric on a particular host.
+
+You can use [variables](#variables) as tag values so they can be configured dynamically
+from the report level.
+
+### Aggregation functions
+
+Back to the example from the section above, if you choose aggregation function instead of `all separately`
+only one data series is created for all the hosts. The result depends on the aggregation function
+function you choose. The aggregation function determines how the metrics are
+aggregated into a single data series. You can pick one of `avg`, `sum`, `min`, `max`
+
+
+## Rollup by
+
+Datapoints sent to Sematext have time label. If a metric is updated every 10 second, and you are
+looking at 24 hours time range, you need 14400 points to display all datapoints. 
+Presenting so many datapoints on the chart would make the chart not readable and would impact
+the web browser performance. If we build time buckets each containing 600 datapoints
+we will have 24 datapoints to present on the chart (each datapoint for each hour). Rollup function
+decides how time buckets are built (what aggregation is performed).
+
+Each of predefined metrics has its own default rollup function defined depending on the type of
```

Comment:
Each of predefined metrics has its own default rollup function defined depending on the type of
=>
Each one of the predefined metrics has its own default rollup function defined depending on the type of